docs/DeveloperGuide.md

```diff
@@ -434,19 +458,24 @@ An instance of `CommandResult` is then created which contains the message and in
 
 Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unlikely to have) - `*`
 
-| Priority | As a …​ | I want to …​                      | So that I can…​                                |
-|----------|---------|-----------------------------------|------------------------------------------------|
-| `* * *`  | user    | view all the available commands   | I know how to use the app                      |
-| `* * *`  | user    | add a new applicant               | track the the progress of all applicants       |
-| `* * *`  | user    | edit an applicant descriptor      | maintain an updated database of all applicants |
-| `* * *`  | user    | view the full list of applicants  | access important applicant information         |
-| `* * *`  | user    | delete an applicant entry         | only track valid applicants                    |
-| `* * *`  | user    | add an interview for an applicant | plan screenings                                |
-| `* * *`  | user    | store data locally                | use it on a daily basis consistently           |
-| `* *`    | user    | find a specific applicant         | access the applicant's information quickly     |
-| `* *`    | user    | sort applicants by a descriptor   | find relevant applicants quickly               |
-| `* *`    | user    | filter applicants by a descriptor | find relevant applicants quickly               |
-| `* *`    | user    | purge all existing data           | remove sample data and populate real data      |
+| Priority | As a …​ | I want to …​                                               | So that I can…​                              |
+|----------|---------|------------------------------------------------------------|----------------------------------------------|
+| `* * *`  | user    | view all the available commands                            | know how to use the app                      |
+| `* * *`  | user    | add a new applicant                                        | track the the progress of all applicants     |
+| `* * *`  | user    | edit an applicant descriptor                               | maintain an updated database of all applicants |
+| `* * *`  | user    | view the full list of applicants                           | access important applicant information       |
```

Comment:
For this maybe you can change the "so that I can.." to "view the overall progress and performance of all applicants". "Access important applicant information" does not really relate to the purpose of the list feature which is to see all the applicants from a bird's eye view